main.scss

```diff
@@ -1,14 +1,183 @@
 @import 'src/scss/functions';
 @import 'src/scss/mixins';
 
+@import 'o-buttons/main';
+@import 'o-fonts/main';
+@import 'o-colors/main';
+@import 'o-typography/main';
+
 @if ($o-comments-is-silent == false) {
-	@include oComments();
+	@include oComments;
 
 	// Set to silent again to avoid being output twice
 	$o-comments-is-silent: true !global;
 
-	.talk-plugin-infobox-info {
-		background: pink;
+	@include oFontsInclude(MetricWeb, $style: italic);
+	@include oFontsInclude(MetricWeb, $weight: semibold, $style: italic);
+	@include oFontsInclude(MetricWeb, $weight: medium, $style: italic);
+
+	@include oFontsInclude(MetricWeb);
+	@include oFontsInclude(MetricWeb, $weight: semibold);
+	@include oFontsInclude(MetricWeb, $weight: medium);
+
+	@include oFontsInclude(FinancierDisplayWeb, $style: italic);
+	@include oFontsInclude(FinancierDisplayWeb, $weight: semibold, $style: italic);
+	@include oFontsInclude(FinancierDisplayWeb, $weight: medium, $style: italic);
+
+	@include oFontsInclude(FinancierDisplayWeb);
+	@include oFontsInclude(FinancierDisplayWeb, $weight: semibold);
+	@include oFontsInclude(FinancierDisplayWeb, $weight: medium);
+
+	:root {
+		--font-family-sans-serif: 'MetricWeb';
+		--font-family-serif: 'FinancierDisplayWeb';
+
+
+		// primary text color
+		--palette-text-primary: #{inspect(oColorsGetPaletteColor('black-80'))};
+
+
+		// error message (ex: edit time has expired)
+		// background
+		--palette-error-light: #{inspect(oColorsGetPaletteColor('claret-80'))};
+		// border-left
+		--palette-error-darkest: #{inspect(oColorsGetPaletteColor('claret-70'))};
+
+
+		// community guidelines box
+		// background
+		--palette-primary-lightest: #{inspect(oColorsGetPaletteColor('wheat'))};
+		// border
+		--palette-primary-light: #{inspect(oColorsGetPaletteColor('wheat'))};
+
+
+		// apply button in configure tab
+		--palette-success-main: #{inspect(oColorsGetPaletteColor('teal-50'))};
+		--palette-success-light: #{inspect(oColorsGetPaletteColor('teal-60'))};
+
+
+		// stream section selected tab item, selected item count badge
+		--palette-primary-main: #{inspect(oColorsGetPaletteColor('teal-50'))};
+
+
+		// stream section unselected tab comment count badge
+		// blockquote background
+		--palette-grey-lightest: #{inspect(oColorsGetPaletteColor('wheat'))};
+
+
+		// stream section tab bar divider
+		--palette-divider: #{inspect(oColorsGetPaletteColor('teal-50'))};
+
+
+		// respect, reply, share, report
+		--palette-grey-main: #{inspect(oColorsGetPaletteColor('teal-50'))};
+
+
+		// report button "reported" state
+		--palette-error-main: #{inspect(oColorsGetPaletteColor('teal-50'))};
+
+
+		// moderation drop down
+		// "Approved" button (approve button state)
+		--palette-success-dark: #{inspect(oColorsGetPaletteColor('black'))};
+		// "Un Feature" button (feature button state)
+		--palette-primary-dark: #{inspect(oColorsGetPaletteColor('black'))};
+	}
+
+	// Mainly for the links in the community guidelines box
+	a {
```

Comment:
Could this anchor style be more specific? I don't know what markup you have to work with but something like`.coral a`? Defensively, so it doesn't impact anything else on the page should o-comments be included outside the iframe?

Reply:
Currently, there is no class assigned to the anchors we target. Therefore we are temporarily targeting a parent div's id to make this more specific.

Ideally, we need to request a class from Coral for the element we are targeting. Will ask them and update this later.